proxy-node-shared/src/main/java/uk/gov/ida/notification/saml/SamlObjectSigner.java

```diff
@@ -17,7 +18,8 @@ public SamlObjectSigner(BasicX509Credential signingCredential, String signingAlg
         signingParams = SignatureSigningParametersHelper.build(signingCredential, signingAlgorithm);
     }
 
-    public void sign(SignableSAMLObject signableSAMLObject) throws MarshallingException, SecurityException, SignatureException {
+    public void sign(SignableSAMLObject signableSAMLObject, String responseId) throws MarshallingException, SecurityException, SignatureException {
+        TranslatorSigningLoggerHelper.logSigningRequest(responseId, signingParams.getSigningCredential().getEntityId());
```

Comment:
Since the signing code and the signing logger both live in the shared code, they're not Translator specific. To make it clear, I'd rename `TranslatorSigningLoggerHelper ` to simply `SamlSigningLogger`. The `logSigningRequest ` method will be invoked every time - regardless of where it's called from - and it may not always come the Translator as it is now so just to clear that up.

Reply:
I tried this and Codacy required Helper on the end of all utility classes.

Reply:
As discussed, we can ignore this rule for now. I'll silence it since it doesn't make sense.

And my comment was more about the `Translator` part of the name - the signing method lives in a shared library and can be called from anywhere - not just from the Translator (even if at the moment that happens to be the only places it's used)

Reply:
Happy to change this name.